src/components/HelloWorld.vue

```diff
@@ -1,32 +1,19 @@
-    </ul>
-    <h3>Essential Links</h3>
-    <ul>
-      <li><a href="https://vuejs.org" target="_blank" rel="noopener">Core Docs</a></li>
-      <li><a href="https://forum.vuejs.org" target="_blank" rel="noopener">Forum</a></li>
-      <li><a href="https://chat.vuejs.org" target="_blank" rel="noopener">Community Chat</a></li>
-      <li><a href="https://twitter.com/vuejs" target="_blank" rel="noopener">Twitter</a></li>
-      <li><a href="https://news.vuejs.org" target="_blank" rel="noopener">News</a></li>
-    </ul>
-    <h3>Ecosystem</h3>
-    <ul>
-      <li><a href="https://router.vuejs.org" target="_blank" rel="noopener">vue-router</a></li>
-      <li><a href="https://vuex.vuejs.org" target="_blank" rel="noopener">vuex</a></li>
-      <li><a href="https://github.com/vuejs/vue-devtools#vue-devtools" target="_blank" rel="noopener">vue-devtools</a></li>
-      <li><a href="https://vue-loader.vuejs.org" target="_blank" rel="noopener">vue-loader</a></li>
-      <li><a href="https://github.com/vuejs/awesome-vue" target="_blank" rel="noopener">awesome-vue</a></li>
-    </ul>
+      <br><br><br><br>
+      <v-btn icon height="10px" width="10px" >
+        <v-avatar>
+          <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAOEAAADhCAMAAAAJbSJIAAAAclBMVEX///8AAABKSko+Pj709PSlpaXV1dXPz8/s7Oy7u7uioqKtra00NDSJiYm2trZxcXHDw8NPT0/b29uWlpZ8fHxbW1sbGxvh4eENDQ2Hh4dpaWn39/fGxsYVFRU3NzchISEqKiouLi5kZGRVVVWSkpJCQkJT6cvnAAAEbklEQVR4nO2da3uiQAxG8QJeQMFbta21Wuv//4tbu31I0h1BtwzpxPd8zE7dHEAIEwajCAAAAAAAAAAAAAAAAAAAAAAAAAiXdTpZHbvBsRo/X6c36YTLoN4vOWon+TPWNX7PT9oZ/pRZzQ7Uzq8BKr+KI+3smmBeITjVTq4RKr6IW+3cmuGyYF87tWZILxsGfpX4Ijd+ltlPK04zD3LsZLvuh0eF37fzaMXBHCwn5vcy1M7GA0O+By0KRikT3Gon44WcBGtK11DpkWGhnYsf2EFaecoNFxLcaKfiCTJ80E7FE+wo1U7FE/a/h/bPpex6eNTOxQ+8pkm0k/ECr0s3100ch8aOKWYmFflhavRsIww7u3hubkfKndg0p3yqf9eZeVU8W1ZNE7XB2rfhx1laef6njUn9rK755ZdxC4od3UM1bkNxrKrYysS3ruKw24LiSFUxKl79K2pfGkdvvg1/wYTlYJFnLx4VlY9TDwynPWG4107IB7KHbvLmpb9nhivtbLwgHhQwd2/2Ca8nbDa4+FTJu3YufmB32W/aufiB3YLutHPxhPErYsQNX7RT8QQMw+eeDFVazan3+8LWeePTl37nvPUo5/aCf0b/Irl1wS9Fq4foX0ZR9Kydg2es78LzTrT8LTyzjHxOHP4Gsqh+TNhszBt27s5wc+oxauMiTHNND1eE6dGBVx4+lOE9C59ovvXQqw+LBtP3J01EPU7xrjsunoOjOVGxPIcmaJY8TMsdRU+RpsdjHqYuvGh7UFtXzFC+Vxhm7pSlIcVFI4meHhOGc7fhoAwveJjmVYU4XbVFp5x686JVIBYzwxCGMIQhDGF4s6H9K779qs0gMAwfGIYPDMMHhuFzd4b7ZOAiYaswOws2JlmW4XHiHJ/yj6FO85SH6QHzgoepOhS5lLP0G2euHyOuvre4UMuK2peqe1H70v8hXk5F20OsLlmVYbEQeUZZcsp3zVxcXC+6TVca8q0iDBe0+Xn4sQyLVam0a2EIQxjCEIYGDe1cDw/rxMVc1DRz9g+0s+K5c/yUh6mmGfFPJ5UtD9M7yHgua6ppnLl+/GHVXJtBYBg+MAwfGIYPDMPn7gz3xYhg5V3G49Su3LmHH3mYKtcVD1PJnvPwxBkuqKB9ZIMLKg7feVgsraiqvNkD0l13/Jou94Ue8IUud1GGRQ+Ytt5tdX+Nock+PgxhCEMYwtCAIV0PfRn2hwRbQn9yx7s8TCpPPEzPceU8TDtrwsJ9UlnwMKnEPExTTlMepgroX0ODwDB8YBg+MAwfGIYPDMPnZ5V3xe+A/SdUgP6OewsY3g4MYfgJDB0mMGwOGDa7/rC+99S8Ia0o9NR7ek1jgr1r/+COd3m4CVJ6Xc4T++yUGoW5e/SSh+k5uX8NDXIHhhvtDDyzET+FZJFetKwfFDSPrENik0EU7etHBcy5hEnqhwXMXBYG9oi/V0rWKF/Sum7jF2ba58iXPAzjfNa1xCxPtX/RBgAAAAAAAAAAAAAAAAAAAAAAALiZPywKm3zqbz9NAAAAAElFTkSuQmCC">
```

Comment:
just wondering, is there a way to shorten this part?

Reply:
perhaps instead of hardcoding the image, you could store it as an asset in the repo and then place the src to that? Such as src="./assets/images/myImg.png"